src/battle_game/core/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ list(APPEND LIBRARY_LIST ${lib_name})
 
 add_library(${lib_name})
 file(GLOB_RECURSE source_files *.cpp *.h)
-target_sources(${lib_name} PRIVATE ${source_files})
+target_sources(${lib_name} PRIVATE ${source_files} "units/laser_tank.cpp" "units/laser_tank.h" "bullets/laser.h" "bullets/laser.cpp" )
```

Comment:
No need to change it, `source_files` will cover these

Reply:
Already changed